src/posix/platform/alarm.c

```diff
@@ -52,7 +52,7 @@ static uint32_t sUsAlarm     = 0;
 static uint32_t sSpeedUpFactor = 1;
 
 #if !OPENTHREAD_POSIX_VIRTUAL_TIME
-uint64_t otSysGetTime(void)
+uint64_t platformGetTime(void)
```

Comment:
Due to #2592, we moved to less generic namespacing in `examples`. Do you think the same should apply to `src/posix`?

Reply:
Apologies for the confusion. After looking at #2592 again, I realize that the comment only applied to APIs used by the apps (i.e. those declared in [examples/platforms/openthread-system.h](https://github.com/openthread/openthread/blob/master/examples/platforms/openthread-system.h)). In light of that, it might be better to stick with what you originally had here. Thoughts?

Reply:
Thanks, I just reverted.